### PR TITLE
[6] Product Page using HTML

### PR DESCRIPTION
**Pull request by kazeemerinfolami**
## Task: `[6] Product Page using HTML`
- `E-Commerce Project`
Screen Shot 2020-09-13 at 7 41 50 PM
Screen Shot 2020-09-13 at 7 41 53 PM

### CODE REVIEW

**Comment by adaugochi** [comment]
Well done @kazeemerinfolami 

But you are expected to show us the image from the web browser and not an image of your code

File: submissions/kazeemErinfolami/aboutMe/html/index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/kazeemErinfolami/aboutMe/html/index.html:1]: This file shouldn't be inside your `aboutMe` folder. Create an `html` folder inside your named folder and place the file inside that folder

Comment (adaugochi) [reply submissions/kazeemErinfolami/aboutMe/html/index.html:1]: Same goes to others as well.

**Review by adaugochi — COMMENTED** [review]
Also, you were asked to use just 2 images, but I can see more than 2 images here.

**Comment by kazeemerinfolami** [comment]
Screen Shot 2020-09-13 at 7 41 53 PM
Screen Shot 2020-09-13 at 7 41 50 PM

**Comment by kazeemerinfolami** [comment]
Screen Shot 2020-09-13 at 7 41 50 PM
Screen Shot 2020-09-13 at 7 41 53 PM

**Review by adaugochi — APPROVED** [review]

**Pull request merged**